cli/src/opts/cast.rs

```diff
@@ -276,12 +276,12 @@ pub enum Subcommands {
         "2r"]
     )]
     ToBase {
-        #[clap(flatten)]
-        base: ToBaseArgs,
-
         /// The output base.
         #[clap(value_name = "BASE")]
         base_out: Option<String>,
+
+        #[clap(flatten)]
+        base: ToBaseArgs,
```

Comment:
This swaps the order of positional arguments, see ci failures

Reply:
Should be fixed now. Went over the other changes, don't think there are any other positional arguments affected